src/main/java/Duke.java

```diff
@@ -1,10 +1,136 @@
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printAcknowledgeMessage(Task task) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task: ");
+        System.out.println("\t[" + task.getType() + "]" + "[✗]" + task.getTaskName());
+        System.out.println("Now you have " + taskCount + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printTaskList(int taskCount, Task[] userTasks) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Here are the tasks in your list:");
+        if (taskCount == 0) {
+            System.out.println("\tThe task is empty. Please add a task");
+        } else {
+            for (int i = 0; i < taskCount; i++) {
+                System.out.println("\t" + (i + 1) + ". " + userTasks[i].toString());
+            }
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printMarkDone(int taskId, Task[] userTasks) {
+        System.out.println("____________________________________________________________");
+        System.out.println("\tNice! I've marked this task as done: \n\t" + userTasks[taskId].toString());
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printHelpMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println("The command you entered is invalid");
+        System.out.println("Enter one of these commands");
+        System.out.println("A. ADD A TASK");
+        System.out.println("\t1. Todo - Task with only the description\n\ttodo <task description>");
+        System.out.println("\t2. Deadline - Task with a set deadline \n\tdeadline <task description> /by <date of completion>");
+        System.out.println("\t3. Event - Task with a set date\n\tevent <task description> /at <date of event>");
+        System.out.println("B. LIST THE TASK");
+        System.out.println("\t list");
+        System.out.println("C. MARK THE TASK AS DONE");
+        System.out.println("\t done <task index>");
+        System.out.println("D. END THE PROGRAM");
+        System.out.println("\t bye");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addTask(Task t) {
+        userTasks[taskCount] = t;
+        taskCount += 1;
+    }
+
+    public static void main(String[] args) {
+        Scanner myObj = new Scanner(System.in);
+        boolean isFinished = false;
+
+        //greets the user with the greeting message
+        printGreetMessaage();
+        while (!isFinished) {
```

Comment:
Maybe you can put the long if/else chain into a switch statement? May improve the code quality